.travis.yml

```diff
@@ -135,7 +135,7 @@ jobs:
   - stage: deploy
     name: Deploy to GCS
     # Only deploy for pushes to master (and if manually invoked)
```

Comment:
Might want to tweak this comment now (wrt need for tag)...  (Although that's pretty much self-evident from the code...)